src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.html

```diff
@@ -26,11 +27,21 @@ <h1 class='slds-text-body_regular' data-qa-locator='heading'>{labels.elevateWidg
                         </div>
                     </lightning-layout-item>
 
+                    <template if:false={isDisabled}>
+                        <lightning-layout-item padding="horizontal-small" size="6">
+                            <lightning-input name="cardholderName"
+                                             label={labels.cardholderNameLabel}
+                                             data-qa-locator="Elevate Cardholder Name"
```

Comment:
data-qa-locator should start with the field type. In this case it will be 'text Elevate Cardholder Name'

Reply:
I can change that, though honestly I'm not sure why it uses the data type. It would make sense if there were more than one field in the UI for the same "label", but that's not generally the case.  I think I questioned that in the Quip doc as well.